package.json

```diff
@@ -30,7 +30,8 @@
     "babel-plugin-transform-react-constant-elements": "^6.9.1",
     "babel-plugin-transform-react-inline-elements": "^6.8.0",
     "babel-plugin-transform-runtime": "^6.9.0",
-    "babel-preset-es2015": "^6.5.0",
+    "babel-preset-es2015": "^6.9.0",
+    "babel-preset-es2015-loose": "^7.0.0",
```

Comment:
meant to remove this, it was part of a hack to get router working since by transpiling object props as non-enumerable http://www.2ality.com/2015/12/babel6-loose-mode.html i will revert this back